source/fundamentals/serialization/poco.txt

```diff
@@ -382,50 +382,86 @@ Specify an ID Generator
 to a collection, if the ``Id`` property contains the default
-value for its data type, the {+driver-short+} doesn't serialize it. Instead, the driver
-generates a unique ID value for the property.
+value for its data type (usually ``null``), the {+driver-short+} doesn't serialize the
+default value. Instead, the driver generates a unique ID value for the property.
 
-If the ``Id`` property is of type ``Guid``, ``ObjectId``, or ``string``, the driver can
-generate the ID value on its own. If the ``Id`` property is any other data type, you must
-specify the ``IIdGenerator`` type that the driver uses to generate the value.
-To specify the ``IIdGenerator`` type, apply the ``[BsonId(IdGenerator)]`` attribute
-to the property and pass the ``IIdGenerator`` type as an argument.
-
-The {+driver-short+} includes the following ``IIdGenerator`` types:
+The following table describes the generators available in the {+driver-short+}:
 
 .. list-table::
    :header-rows: 1
-   :widths: 10 10
+   :widths: 25 75
 
    * - ``Id`` Field Data Type
-     - ``IIdGenerator`` Type
-   * - ``Guid`` value generated by the COMB algorithm
-     - ``CombGuidGenerator``
-   * - ``BsonObjectId``
-     - ``BsonObjectIdGenerator``
+     - How to Use
 
-In the following code example, if the ``Id`` property of the ``House`` class contains
-the default value (``null``), the driver uses the COMB algorithm to generate a
-unique value during serialization:
+   * - ``Guid``
+     - To use the COMB algorithm to generate a unique ``Guid`` value, apply the
+       ``[BsonId(IdGenerator = typeof(CombGuidGenerator))]`` attribute to the ``Id``
+       property, as shown in the following example:
 
-.. code-block:: csharp
-   :copyable: false
-   :emphasize-lines: 3
-   
-   public class House
-   {
-       [BsonId(IdGenerator = typeof(CombGuidGenerator))]
-       public Guid Id { get; set; }
-   }
+       .. code-block:: csharp
+          :copyable: true
 
-.. note::
-   
-   In the previous code example, if the ``Id`` property didn't have the
-   ``[BsonId(IdGenerator)]`` attribute, the driver would generate a non-COMB GUID
-   to assign to the ``Id`` field.
+          public class House
+          {
+             [BsonId(IdGenerator = typeof(CombGuidGenerator))]
+             public Guid Id { get; set; }
+          }
+
+       To generate a unique ``Guid`` value without the COMB algorithm, don't apply
+       the preceding attribute to the ``Id`` property. The driver automatically uses
+       the ``GuidGenerator`` type to generate a unique value for the ``Id`` field.
+
+       .. code-block:: csharp
+          :copyable: true
+
+          public class House
+          {
+             public Guid Id { get; set; }
+          } 
+
+   * - ``ObjectId``
+     - The driver automatically uses the ``ObjectIdGenerator`` type for ``Id`` fields with
+       the ``ObjectId`` data type, like the one in the following code example:
```

Comment:
https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#std-term-like
```suggestion
       the ``ObjectId`` data type, such as the one in the following code example:
```

Reply:
boo this busted rule